cpp/possiblePath.cpp

```diff
@@ -0,0 +1,23 @@
+//https://www.hackerrank.com/challenges/possible-path/
```

Comment:
__nitpick__ when writing comments in code, always leave a space after the comment sign `//` so it should be like 
`//start of comm..` --> `// start of comm..` also another thing, when writing a url path, the url path must never end with  the `/` symbol. thats a like a no-no in web development.

so it should be
```
https://www.hackerrank.com/challenges/possible-path
``` 
instead of 
```
https://www.hackerrank.com/challenges/possible-path/
```

I know that these are very small things but they are what differentiate us from the crowd 😉